### PR TITLE
Fill in todo annotations

### PR DESCRIPTION
**Pull request by NotMyFault**
```
➜  jenkins git:(todo-update) ./update-since-todo.py 
Analyzing core/src/main/java/jenkins/model/ModelObjectWithContextMenu.java:153
        first sha: 291f5edcc5942c4efc65cc96121f7624f45bf3c3
        first tag was jenkins-2.401
        Updating file in place
Analyzing core/src/main/java/jenkins/model/ModelObjectWithContextMenu.java:365
        first sha: 291f5edcc5942c4efc65cc96121f7624f45bf3c3
        first tag was jenkins-2.401
        Updating file in place
Analyzing core/src/main/java/jenkins/model/lazy/LazyBuildMixIn.java:273
        first sha: fe52887da2fc8810e8a5bffcdda95c6698cef7be
        first tag was jenkins-2.407
        Updating file in place
Analyzing core/src/main/java/jenkins/model/queue/QueueItem.java:14
        first sha: cce017cdecd04e506778378ec79523239404c065
        first tag was jenkins-2.405
        Updating file in place
Analyzing core/src/main/java/jenkins/util/HttpServletFilter.java:48
        first sha: b560aa455c055cb41f4d7cd9b2088a41594a79cc
        first tag was jenkins-2.406
        Updating file in place
Analyzing core/src/main/java/jenkins/widgets/HasWidgets.java:40
        first sha: 65d6007593868be1756e7c5b58c225bd7c43d4a1
        first tag was jenkins-2.410
        Updating file in place
Analyzing core/src/main/java/jenkins/widgets/WidgetFactory.java:42
        first sha: 65d6007593868be1756e7c5b58c225bd7c43d4a1
        first tag was jenkins-2.410
        Updating file in place
Analyzing core/src/main/java/hudson/model/Job.java:618
        first sha: 95840c1fb7453db7658822092df11f2b26d42ed5
        first tag was jenkins-2.410
        Updating file in place
```

List of commits introducing new API and the first release they went in:
* https://github.com/jenkinsci/jenkins/releases/tag/jenkins-2.401
  - https://github.com/jenkinsci/jenkins/commit/291f5edcc5942c4efc65cc96121f7624f45bf3c3
* https://github.com/jenkinsci/jenkins/releases/tag/jenkins-2.405
  - https://github.com/jenkinsci/jenkins/commit/cce017cdecd04e506778378ec79523239404c065
* https://github.com/jenkinsci/jenkins/releases/tag/jenkins-2.406
  - https://github.com/jenkinsci/jenkins/commit/b560aa455c055cb41f4d7cd9b2088a41594a79cc
* https://github.com/jenkinsci/jenkins/releases/tag/jenkins-2.407
  - https://github.com/jenkinsci/jenkins/commit/fe52887da2fc8810e8a5bffcdda95c6698cef7be
* https://github.com/jenkinsci/jenkins/releases/tag/jenkins-2.410
  - https://github.com/jenkinsci/jenkins/commit/65d6007593868be1756e7c5b58c225bd7c43d4a1
  ```

<a href="https://gitpod.io/#https://github.com/jenkinsci/jenkins/pull/8209"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>

### CODE REVIEW

**Review by basil — APPROVED** [review]

**Review by daniel-beck — APPROVED** [review]

**Comment by daniel-beck** [comment]
This PR is now ready for merge. We will merge it after approximately 24 hours if there is no negative feedback. Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process. Thanks!

**Comment by daniel-beck** [comment]
Will merge once the PR build passed.

**Pull request merged**
